.github/workflows/test.yml

```diff
@@ -37,6 +37,11 @@ jobs:
         with:
           command: test
 
+      - uses: actions-rs/cargo@v1
+        with:
+          command: bench
+          args: --no-run -p benchmarks
+
```

Comment:
can't we modify the cargo build action L32 by adding `args` under the `command` in order to check all packages?
```yaml
      - uses: actions-rs/cargo@v1
        with:
              command: build
              args: # ...
```

Reply:
Yup we could but add this too!

Reply:
I just added a `check --all` CI job.

Reply:
If we already have a `cargo check --all` what is the purpose of an additional `cargo bench --no-run`?

Reply:
That's because `cargo bench` compiles the benchmarks which `cargo check` don't.